prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,1155 @@
+      "Description": "A pre-established collection of rules under management control.",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Must"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "Global allow - and block-lists",
+      "Description": "Establishing International Lists for Permissions and Restrictions",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Must"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "Blanket rate-based rules",
+      "Description": "Establishing rules based on a standardized, all-encompassing rate.",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Must"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "Service-unique exclusion rules",
+      "Description": "Exclusion rules specific to the service provided.",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Must"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "Additional managed rules",
+      "Description": "Supplementary managed rules",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Discuss"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "Scoped-down rate-based rules",
+      "Description": "Rate-based rules with a narrowed scope",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Deploy WAF setup for each public web service",
+          "Service": "WAF",
+          "Type": "Discuss"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "For first 2 weeks - in COUNT mode",
+      "Description": "During the initial two-week period, operating in COUNT mode.",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Enable Shield Advanced automatic application layer DDoS mitigation",
+          "Service": "Shield",
+          "Type": "Manual"
+        }
+      ],
+      "Checks": []
+    },
+    {
+      "Id": "After that - switch into BLOCK mode",
+      "Description": "Following that, transition into BLOCK mode.",
+      "Attributes": [
+        {
+          "Section": "WAFv2",
+          "SubSection": "Enable Shield Advanced automatic application layer DDoS mitigation",
+          "Service": "Shield",
+          "Type": "Manual"
+        }
+      ],
+      "Checks": []
+    },
```

Comment:
This is a single one "Enable Shield Advanced automatic application layer DDoS mitigation"

Reply:
Done!